presidio-anonymizer/presidio_anonymizer/batch_anonymizer_engine.py

```diff
@@ -0,0 +1,83 @@
+import collections
+from typing import List, Dict, Union, Iterable
+
+from presidio_anonymizer import AnonymizerEngine
+from presidio_anonymizer.entities import DictRecognizerResult
+from presidio_anonymizer.entities import EngineResult, RecognizerResult
+
+
+class BatchAnonymizerEngine:
+    """
+    BatchAnonymizerEngine class.
+
+    A class that provides functionality to anonymize in batches.
+    """
+
+    def __init__(self, anonymizer_engine: AnonymizerEngine = None):

```

Comment:
```suggestion
    def __init__(self, anonymizer_engine: Optional[AnonymizerEngine] = None):

```

If you accept, please add `Optional` to the imports.